.github/workflows/edocs-synchronization.yml

```diff
@@ -0,0 +1,24 @@
+name: Synchronize with eDocs
+on:
+  workflow_dispatch:
+    inputs:
+      branchName:
+        description: 'Branch origin'
+        required: true
+      version:
+        description: 'Version'
+        required: true
+
+jobs:
+  connect-with-edocs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Calling to edocs action...
+        run: |
+          curl --request POST \
+          -u empathyco:${GITHUB_TOKEN} \
+          --header 'content-type: application/json' \
+          --url https://api.github.com/repos/empathyco/docs-empathy-documentation/actions/workflows/14218449/dispatches \
```

Comment:
Instead of hardcoded the URL, maybe we can use an [ENV variable](https://docs.github.com/en/actions/learn-github-actions/environment-variables), what do you think?